### PR TITLE
Renaming core.pyx => core.py

### PR DESCRIPTION
**Pull request by madsbk**
Renaming `core.pyx` => `core.py`. No change to behavior.

### CODE REVIEW

File: tests/test_guarantee_msg_order.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def test_mismatch(server_guarantee_msg_order):
 
     # We use an exception handle to catch errors raised by the server
     def handle_exception(loop, context):
-        msg = context.get("exception", context["message"])
+        msg = str(context.get("exception", context["message"]))
```
Comment (jakirkham) [inline tests/test_guarantee_msg_order.py:12]: Should we change how `context["exception"]` and `context["message"]` are set to ensure they are `stress?

Comment (madsbk) [reply tests/test_guarantee_msg_order.py:12]: I am not sure if that is possible. It is defined here: https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.set_exception_handler

Comment (jakirkham) [reply tests/test_guarantee_msg_order.py:12]: Oh in that case wouldn't they already be `str`s?

Comment (madsbk) [reply tests/test_guarantee_msg_order.py:12]: `context["exception"]` is a `ValueError`. I use `str()` to assert the message of the error.

**Comment by pentschev** [comment]
@rapidsai/ops-codeowners I think you may need to approve this due to the change in `.flake8`. Could you check?

**Pull request merged**